helmfile/charts/gatekeeper-templates/policies/disallow-tag.rego

```diff
@@ -0,0 +1,84 @@
+package k8sdisallowedtags
+
+# For regular containers
+violation[{"msg": msg}] {
+
+    container := get_containers[_]
+
+    tags := [forbid | tag = input.parameters.tags[_] ; forbid = endswith(container.image, concat(":", ["", tag]))]
+    any(tags)
+    msg := sprintf("container <%v> uses a disallowed tag <%v>; disallowed tags are %v", [container.name, container.image, input.parameters.tags])
+}
+
+violation[{"msg": msg}] {
+
+    container := get_containers[_]
+
+    tag := [contains(container.image, ":")]
+    not all(tag)
+    msg := sprintf("container <%v> didn't specify an image tag <%v>", [container.name, container.image])
+}
```

Comment:
You can add this check for init-containers as well.

Reply:
Yes , i added this checks as well .